features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -1157,7 +1169,11 @@
     </Cookies-->
 
 
+    {% if resource_access_control.default_access is sameas allow %}
```

Comment:
Is the value of the key "resource_access_control.default_access" is allow?
Can we change it to true or false? 
"resource_access_control.allow_access" : true

Reply:
In the xml configuration, it should be either `<ResourceAccessControl default-access="deny">` for deny and `<ResourceAccessControl>` for previous behavior (allow). So i think "allow" and "deny" are more appropriate. WDYT?